rotkehlchen/chain/ethereum/types.py

```diff
@@ -85,3 +85,6 @@ def deserialize(
             weight=FVal(data['weight']) / 100,
             active=bool(data['active']),
         )
+
+
+ERC20_INFO_METHODS = Tuple[str, str, str]
```

Comment:
This is just used in one place. It's really overengineering. Just use`Tuple[str, str, str,]` there directly